### DISCUSSION TITLE
dependsOn from a root task

### DISCUSSION THREAD

**Discussion by nicotsx** [vercel/turborepo#1745] [Help] [answered]
Hello, I cannot find in the documentation or in issues how I can make a root task `//#` depend on a workspace task.
For example :
```
{
  "$schema": "https://turborepo.org/schema.json",
  "pipeline": {
    "build": {
      "dependsOn": ["^build"],
      "outputs": ["dist/**"]
    },
    "test": {
      "dependsOn": ["build"],
      "outputs": ["coverage/**"],
      "inputs": ["src/**/*.tsx", "src/**/*.ts", "test/**/*.ts", "test/**/*.tsx"]
    },
    "lint": {
      "outputs": [],
      "inputs": ["src/**/*.tsx", "src/**/*.ts"]
    },
    "//#coverage:combined": {
      "dependsOn": ["test"],
      "outputs": ["coverage/**"]
    }
  }
}
```

How can I make `//#coverage:combined` depend on `test` ? Currently I get an error : error preparing engine: //#test needs an entry in turbo.json before it can be depended on because it is a task run from the root package

**Comment by mehulkar** [comment, marked as answer]
This looks like a bug. Could you open an issue and provide a reproduction if possible? Thank you!
